tpcc/main.go

```diff
@@ -337,17 +339,22 @@ func main() {
 				cumLatency.Merge(m)
 			}
 
-			ops := atomic.LoadUint64(&txs[newOrderType].numOps)
-			elapsed := time.Since(start).Seconds()
-			fmt.Println("\n_elapsed___newOrders___tpmC(cum)__avg(ms)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)")
-			fmt.Printf("%7.1fs %11d %11.1f %8.1f %8.1f %8.1f %8.1f %8.1f\n\n",
-				time.Since(start).Seconds(),
-				ops, float64(ops)/elapsed*60,
-				time.Duration(cumLatencyByOp[newOrderType].Mean()).Seconds()*1000,
-				time.Duration(cumLatencyByOp[newOrderType].ValueAtQuantile(50)).Seconds()*1000,
-				time.Duration(cumLatencyByOp[newOrderType].ValueAtQuantile(95)).Seconds()*1000,
-				time.Duration(cumLatencyByOp[newOrderType].ValueAtQuantile(99)).Seconds()*1000,
-				time.Duration(cumLatencyByOp[newOrderType].ValueAtQuantile(100)).Seconds()*1000)
+			fmt.Println("\n_elapsed______opName__ops(total)__ops/m(cum)__avg(ms)__p50(ms)__p90(ms)__p95(ms)__p99(ms)_pMax(ms)")
+			totalTime := time.Since(start).Seconds()
+			for i, h := range cumLatencyByOp {
+				ops := atomic.LoadUint64(&txs[i].numOps)
+				elapsed := time.Since(start).Seconds()
```

Comment:
NYC but just noticed that this should be pulled outside of the loop or else each op will get a slightly different elapsed time.

Reply:
Heh, this is the same value as `totalTime`. Fixed.